db/models/emailhash.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+module.exports = (sequelize, DataTypes) => {
+  const EmailHash = sequelize.define("EmailHash", {
+    sha1: DataTypes.STRING,
+  }, {});
+
+  EmailHash.associate = function(models) {
+    EmailHash.belongsToMany(models.Breach, { through: "BreachedHashes" });
```

Comment:
The `BreachedHashes` model will need something like the `BreachedUser.notified` flag to keep track of which breached hashes we've notified already.

Reply:
BreachedHashes isn't really a model, it's just the name of the table that is used to store the association data. I think we should add the notified flag to the Subscriber model.

Reply:
I don't understand how to put the notified flag on BreachedHashes. BreachedHashes is simply a table that stores the association between EmailHash and Breach - Sequelize is just letting us name it what we want, and I chose BreachedHashes. And indeed, `console.log(models.BreachedHashes)` outputs `undefined`.

Reply:
We would define a model for `BreachedHashes` and include the additional attribute. It's the part of [the belongs-to-many docs]http://docs.sequelizejs.com/manual/tutorial/associations.html#belongs-to-many-associations) that starts with "If you want additional attributes in your join table ..."

Reply:
I've also forgotten why we moved the `sha1` field out of `Subscriber` and into its own model? :/ In any case, I'll work on top of all this in my branch to finish the templates. We can always change models anytime before we deploy.